test/sass/typograpy.spec.scss

```diff
@@ -159,3 +164,118 @@
   }
 
 }
+
+@include describe('@mixin label-as-content') {
+
+  @include it('generates defaults for label-as-content') {
+
+    @include assert() {
+
+      @include output {
+        @include typography.label-as-content();
+      }
+
+      @include expect {
+        color: $text-secondary;
```

Comment:
Adding local but exportable and overridable properties like this could be misleading and dangerous. Should either specify the color directly like this: `color: rgb(102, 102, 102)`, or use the variable from `_color.scss` like this: `color: color.$text-secondary`. As every other rule value in this test is a specified directly (e.g. the font family), then the colour should follow the same pattern and also be specified directly.